### PR TITLE
Auto-update Changes to move Resource Uris to server for updates

### PR DESCRIPTION
**Pull request by mbhavi**
Auto-update Changes to move Resource Uris to server for updates.

ping @swmitra @vickramdhawal @nethip  for review.

### CODE REVIEW

File: src/extensions/default/AutoUpdate/main.js [hunk 1]
```diff
@@ -421,15 +378,35 @@ define(function (require, exports, module) {
      * @param {Array} updates - array object containing info about updates
```
Comment (swmitra) [inline src/extensions/default/AutoUpdate/main.js]: @mbhavi Please create a JSDoc custom node definition for update entry.

File: src/extensions/default/AutoUpdate/main.js [hunk 2]
```diff
@@ -421,15 +378,35 @@ define(function (require, exports, module) {
...
-        updateParams.latestBuildNumber = updates[0].buildNumber;
-        updateParams.checksum = checksum;
+
+        var platform = getPlatformInfo(),
+            checksum,
+            downloadURL,
+            installerName,
+            assetInfo;
+
+        assetInfo = updates[0].assetInfo;
```
Comment (swmitra) [inline src/extensions/default/AutoUpdate/main.js]: Retain the `updates[0]` in a local var for once upfront.

File: src/extensions/default/AutoUpdate/main.js (hunk 1 above)
Comment (mbhavi) [reply src/extensions/default/AutoUpdate/main.js]: @swmitra , addressed in 44fef7e

File: src/extensions/default/AutoUpdate/main.js (hunk 2 above)
Comment (mbhavi) [reply src/extensions/default/AutoUpdate/main.js]: @swmitra , addressed in 44fef7e

**Review by swmitra — APPROVED** [review]

**Pull request merged**